website/content/docs/concepts/policies.mdx

```diff
@@ -800,32 +800,36 @@ authenticated user.
+| [auth/token/accessors](/vault/api-docs/auth/token#list-accessors)                                                                                      | LIST              | List token accessor                                                                                                                                 |
+| [auth/token/create](/vault/api-docs/auth/token#create-token)                                                                                           | POST              | Create a periodic or an orphan token (`period` or `no_parent`) option                                                                               |
+| [pki/root](/vault/api-docs/secret/pki#delete-all-issuers-and-keys)                                                                                     | DELETE            | Delete the current CA key ([pki secrets engine](/vault/docs/secrets/pki))                                                                           |
+| [pki/root/sign-self-issued](/vault/api-docs/secret/pki#sign-self-issued)                                                                               | POST              | Use the configured CA certificate to sign a self-issued certificate ([pki secrets engine](/vault/docs/secrets/pki))                                 |
 | [sys/audit](/vault/api-docs/system/audit)                                                                                                              | GET               | List enabled audit devices                                                                                                                          |
-| [sys/audit/:path](/vault/api-docs/system/audit)                                                                                                        | PUT, DELETE       | Enable or remove an audit device                                                                                                                    |
+| [sys/audit/:path](/vault/api-docs/system/audit)                                                                                                        | POST, DELETE      | Enable or remove an audit device                                                                                                                    |
 | [sys/auth/:path](/vault/api-docs/system/auth)                                                                                                          | GET, POST, DELETE | Manage the auth methods (enable, read, delete, and tune)                                                                                            |
+| [sys/auth/:path/tune](/vault/api-docs/system/auth#tune-auth-method)                                                                                    | GET, POST, DELETE | Manage the auth methods (enable, read, delete, and tune)                                                                                            |
```

Comment:
Verbs: No DELETE on this endpoint

Description: e.g. `Read/update auth method tuning`

Reply:
Removed delete in upcoming commit.